app/api/hooks.ts

```diff
@@ -177,7 +177,7 @@ export const getUsePrefetchedApiQuery =
       `Expected query to be prefetched.
 Key: ${JSON.stringify(queryKey)}
 Ensure the following:
-• loader is running
+• loader is called in routes.tsx and is running
```

Comment:
haha this tells a story

Reply:
Haha, maybe! 
* puts hands in pockets; whistles nonchalantly *